Index: h -u origin switch
===================================================================
diff --git a/h -u origin switch b/h -u origin switch
new file mode 100644
--- /dev/null	
+++ b/h -u origin switch	
@@ -0,0 +1,34 @@
+diff.astextplain.textconv=astextplain
+filter.lfs.clean=git-lfs clean -- %f
+filter.lfs.smudge=git-lfs smudge -- %f
+filter.lfs.process=git-lfs filter-process
+filter.lfs.required=true
+http.sslbackend=openssl
+http.sslcainfo=D:/Git/mingw64/ssl/certs/ca-bundle.crt
+core.autocrlf=true
+core.fscache=true
+core.symlinks=false
+pull.rebase=false
+credential.helper=manager-core
+credential.https://dev.azure.com.usehttppath=true
+init.defaultbranch=master
+user.name=wuyang77
+user.email=2388958622@qq.com
+safe.directory='E:/自己的项目练习文件/qfshop'
+safe.directory='E:/自己的项目练习文件/qfshop'
+safe.directory=*
+http.sslverify=false
+core.repositoryformatversion=0
+core.filemode=false
+core.bare=false
+core.logallrefupdates=true
+core.symlinks=false
+core.ignorecase=true
+remote.ssm_220106.url=https://github.com/wuyang77/ssm220106.git
+remote.ssm_220106.fetch=+refs/heads/*:refs/remotes/ssm_220106/*
+branch.master.remote=ssm_220106
+branch.master.merge=refs/heads/master
+remote.gitee_ssm220106.url=https://gitee.com/wu-yang77/ssm220106.git
+remote.gitee_ssm220106.fetch=+refs/heads/*:refs/remotes/gitee_ssm220106/*
+branch.switch1.remote=gitee_ssm220106
+branch.switch1.merge=refs/heads/switch1
